src/screens/ENSSearchSheet.tsx

```diff
@@ -24,6 +24,7 @@ import {
 } from '@rainbow-me/design-system';
```

Comment:
My general impression is that it could be broken down into smaller dumb components to avoid so many ternary operators etc